discovery-client/src/main/java/io/micronaut/discovery/consul/registration/ConsulAutoRegistration.java

```diff
@@ -138,7 +138,7 @@ protected void pulsate(ServiceInstance instance, HealthStatus status) {
                             LOG.debug("Successfully reported failure state to Consul");
                         }
                     } else {
-                        String errorMessage = getErrorMessage(throwable, "Error reporting passing state to Consul: ");
+                        String errorMessage = getErrorMessage(throwable, "Error reporting failure state to Consul: ");
```

Comment:
also imho the error message could directly inside `LOG.error` (same for passing), or is there some other reason why the error message is composed but in case debug is not enabled , unused?

Reply:
Seems like a worthwhile improvement as well